### DISCUSSION TITLE
.de Domain Support

### DISCUSSION THREAD

**Discussion by herby2212** [vercel/community#6504] [Feature Question] [answered]
### Question

Greetings,
currently it is not possible to transfer a .de domain over to vercel. Is there an eta when this will be available? As I would like to switch hosting to vercel, but my site uses a .de domain and chaning the domain to for example .com is not an option.

**Comment by amyegan** [comment, marked as answer]
Hey @herby2212. We don't currently support purchasing `.de` domains directly. However, you can still buy from a third party service and use it with your Vercel project using these instructions: https://vercel.com/docs/projects/domains/add-a-domain

**Reply by herby2212** [reply]
Thanks, I will give it a try!

**Reply by herby2212** [reply]
Can confirm switch to vercel with the .de domain worked well using the Nameserver method.
